database/sampleDataMaker.js

```diff
@@ -25,7 +25,7 @@ const createDataObj = () => {
 
 let dataObj = createDataObj();
 
-console.log('dataObj:', dataObj);
+console.log('dataObj = ', dataObj);
```

Comment:
Are these console.logs necessary or do you just need them for debugging or printing prompts?  You could use logger module for that

Reply:
I used it for testing and see the actual structure of the object